src/Artistic-2.0.xml

```diff
@@ -0,0 +1,196 @@
+<spdx name="Artistic License 2.0" identifier="Artistic-2.0" osi-approved="true">
+  <urls>
+    <url>http://www.perlfoundation.org/artistic_license_2_0 </url>
+    <url>http://www.opensource.org/licenses/artistic-license-2.0</url>
+  </urls>
+  <notes>This license was released: 2006</notes>
+  <license>
+    <title>
+      <p>The Artistic License 2.0</p>
+    </title>
+    <copyright>
+      <p>Copyright (c) 2000-2006, The Perl Foundation.</p>
+    </copyright>
+    <optional>
+      <p>Everyone is permitted to copy and distribute verbatim copies of this license document, but changing it is
+         not allowed.</p>
+      <p>Preamble</p>
+      <p>This license establishes the terms under which a given free software Package may be copied, modified,
+         distributed, and/or redistributed. The intent is that the Copyright Holder maintains some artistic
+         control over the development of that Package while still keeping the Package available as open source
+         and free software.</p>
+      <p>You are always permitted to make arrangements wholly outside of this license directly with the Copyright
+         Holder of a given Package. If the terms of this license do not permit the full use that you propose to
+         make of the Package, you should contact the Copyright Holder and seek a different licensing
+         arrangement.</p>
+    </optional>
+    <body>
+      <p>Definitions</p>
```

Comment:
I think that would be perfectly valid. The automated process based the list tags on the presence of a bullet, which is why it was not automatically converted here.